docs/resources/on_call_schedule.md

```diff
@@ -119,3 +119,11 @@ The `restrictions` block supports:
 In addition to all arguments above, the following attributes are exported:
 
 * `id` - The ID of the on-call schedule.
+
+## Import
+
+On-call schedules can be imported; use `<TeamID>:<ScheduleID>` as the import ID. For example:
+
+```shell
+terraform import firehydrant_rotation.example_rotation 3638b647-b99c-5051-b715-eda2c912c42e:12345678-90ab-cdef-1234-567890abcdef
```

Comment:
nit: using the rotation resource/example here

Reply:
Dammit.  Yep.  Fixing this too.